package.json

```diff
@@ -19,11 +19,13 @@
   },
   "author": "Developer Experience <devx@theguardian.com>",
   "dependencies": {
-    "@actions/core": "^1.5.0"
+    "@actions/core": "^1.5.0",
+    "node-fetch": "2.6.2"
```

Comment:
The latest version of `node-fetch` is `3.0.0` and comes included with type declarations. However it has also moved to ESM format and as we're compiling to CommonJS, we'd have to do a bit of [dancing](https://github.com/node-fetch/node-fetch/blob/main/docs/v3-UPGRADE-GUIDE.md) to work with it.

It's simpler to stick to v2 for now, at the cost of needing `@types/node-fetch`.